packages/api/lib/publishSnsMessageUtils.js

```diff
@@ -29,15 +29,38 @@ const constructSnsMessage = (record, event) => {
   }
 };
 
+const constructGranuleSnsMessage = (record, event) => {
+  switch (event) {
+  case 'Create':
+  case 'Update':
+    return { event, record };
+  case 'Delete': return {
+    event,
+    record,
+    deletedAt: Date.now(),
+  };
+  default: return {};
+  }
+};
+
+const publishGranuleSnsMessage = async (record, event) => {
+  const topicArn = envUtils.getRequiredEnvVar('granule_sns_topic_arn', process.env);
+  const messageToPublish = constructGranuleSnsMessage(record, event);
+
+  logger.info(`About to publish SNS message ${JSON.stringify(record)} for granule to topic ARN ${topicArn}`);
+  await publishSnsMessage(topicArn, messageToPublish);
+};
+
```

Comment:
This will most likely change once I merge https://github.com/nasa/cumulus/pull/2449/files#diff-f6424c54f7e3c57a3c734bcba34d76bcf4bbb6de70354f1843109ad34b4a26b1R26